galley/pkg/config/analysis/analyzers/annotations/annotations.go

```diff
@@ -82,8 +84,12 @@ outer:
 
 		annotationDef := lookupAnnotation(ann)
 		if annotationDef == nil {
-			ctx.Report(collectionType,
-				msg.NewUnknownAnnotation(r, ann))
+
+			m := msg.NewUnknownAnnotation(r, ann)
+			line := util.ErrorLineForAnnotation(ann, r)
+			m.SetLine(line)
```

Comment:
Just export the Line property and set it directly, there's no need to use a setter function.